packages/builder/tailwind.config.js

```diff
@@ -44,6 +44,15 @@ module.exports = {
     fontFamily: {
       sans: ['"Libre Franklin"'],
     },
+    typography: (theme) => ({
+      DEFAULT: {
+        css: {
+          'p': {
+            paddingTop: '8px',
```

Comment:
I would avoid having a global padding top for all the `p` tags, lets add it specifically where we need it. 
And in general, do we really need a padding top here, or better having a padding top for the container and maring bottom for each paragraph?